.github/workflows/coverity.yml

```diff
@@ -0,0 +1,28 @@
+name: Coverity scan
+on:
+  # run once daily at 00:30 UTC due to
+  # https://scan.coverity.com/faq#frequency
+  schedule:
+    - cron: "30 0 * * *"
+
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
```

Comment:
Please, deindent list. That will make it optically at the same level as objects.

```yml
key:
  key: value
  key: value
key:
- item
- item
```